Backend/app/rider-platform/rider-app/Main/spec/Storage/MultiModal.yaml

```
@@ -368,6 +368,16 @@ JourneyLeg:
       where:
         and: [journeyId, sequenceNumber]
+    updateEstimatedFaresByJourneyIdAndSequenceNumber:
+      kvFunction: updateWithKV
+      params: [estimatedMinFare, estimatedMaxFare]
+      where:
+        and: [journeyId, sequenceNumber]
+    updateEstimatedFaresBySearchId:
+      kvFunction: updateWithKV
+      params: [estimatedMinFare, estimatedMaxFare]
+      where: searchRequestId
+      where: legSearchId
```

YAMLlint (1.37.1)

[error] duplication of key "where" in mapping

(key-duplicates)